### PR TITLE
New Routes Created

### PR DESCRIPTION
**Pull request by ekottifiok**
Created the dashboard route and the laboratory routes

### CODE REVIEW

**Review by praise-cloud — APPROVED** [review]
approve the merge

**Review by praise-cloud — APPROVED** [review]

**Review by praise-cloud — APPROVED** [review]

**Comment by ukane-philemon** [comment]
@ekottifiok, I see unresolved conflicts in this PR.

**Comment by praise-cloud** [comment]
it is fro my end sir

**Review by praise-cloud — APPROVED** [review]

File: admindashboard/src/components/styled/drawer.ts
Comment (praise-cloud) [inline admindashboard/src/components/styled/drawer.ts:1]: why was this file commented out

**Review by praise-cloud — APPROVED** [review]
Reviewed and approved

File: admindashboard/src/components/styled/drawer.ts
Comment (ekottifiok) [reply admindashboard/src/components/styled/drawer.ts:1]: It should be removed, was testing the best place to put certain things

**Pull request merged**